lib/flutter_periculum.dart

```diff
@@ -68,4 +69,46 @@ class FlutterPericulum {
 
     return jsonEncode(myresponse).toString();
   }
+
```

Comment:
Also please update the affordability call to also return an Affordability model.